invenio_rdm_migrator/extract/transactions.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration transaction extract classes."""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class Tx:
+    """An extracted DB transaction."""
+
+    id: int
+    operations: list[dict]  # TODO: we could more narrowly define it later
```

Comment:
should we clean TODO/FIXME and made the issues before merging? the code base of this module has already 14 of these and growing 😓

Reply:
Agree, let's make an issue out of it and I can start with this task once we merge this one.

Reply:
Opened https://github.com/inveniosoftware/invenio-rdm-migrator/issues/121